src/navigation/ExposureHistoryStack.tsx

```diff
@@ -20,24 +18,6 @@ const SCREEN_OPTIONS = {
 }
 
 const ExposureHistoryStack: FunctionComponent = () => {
-  const navigation = useNavigation()
-  const { observeExposures } = useExposureContext()
```

Comment:
should we also remove this from the context?

Reply:
I think there is a lot of cleanup that can be done here to remove unused code, but I'll open another PR for that.